### PR TITLE
Adding sig_term_handled and zombie_handled to the docs

### PR DESCRIPTION
**Pull request by martin-mat**
Ref: #1924
Ref: #1925

## Description
(name of issue/change)

## Issues:
Refs: #NNN

## How has this been tested:
 - [ ] Covered by existing integration testing
 - [ ] Added integration testing to cover
 - [ ] Verified all A/C passes
     * [ ] develop
     * [ ] master
     * [ ] tag/other branch
 - [ ] Test environment
    * [ ] Shared Packet K8s cluster
    * [ ] New Packet K8s cluster
    * [ ] Kind cluster
 - [ ] Have not tested

## Types of changes:
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [ ] Documentation update

## Checklist:
**Documentation**
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] No updates required.

**Code Review**
- [ ] Does the test handle fatal exceptions, ie. rescue block

**Issue**
- [ ] Tasks in issue are checked off

### CODE REVIEW

File: RATIONALE.md [hunk 1]
```diff
@@ -82,6 +82,14 @@ it forces the two services to upgrade in lock step
...
+
+> The Linux kernel handles signals differently for the process that has PID 1 than it does for other processes. Signal handlers aren't automatically registered for this process, meaning that signals such as SIGTERM or SIGINT will have no effect by default. By default, one must kill processes by using SIGKILL, preventing any graceful shutdown. Depending on the application, using SIGKILL can result in user-facing errors, interrupted writes (for data stores), or unwanted alerts in a monitoring system.
+
+#### *To check if the CNF PID 1 processes handle zombie processes correctly*: [zombie_handled](docs/LIST_OF_TESTS.md#zombie-handled)
+
+> Classic init systems such as systemd are also used to remove (reap) orphaned, zombie processes. Orphaned processesi — processes whose parents have died - are reattached to the process that has PID 1, which should reap them when they die. A normal init system does that. But in a container, this responsibility falls on whatever process has PID 1. If that process doesn't properly handle the reaping, you risk running out of memory or some other resources.
```
Comment (agentpoyo) [inline RATIONALE.md:91]:
```suggestion
> Classic init systems such as systemd are also used to remove (reap) orphaned, zombie processes. Orphaned processes — processes whose parents have died - are reattached to the process that has PID 1, which should reap them when they die. A normal init system does that. But in a container, this responsibility falls on whatever process has PID 1. If that process doesn't properly handle the reaping, you risk running out of memory or some other resources.
```

File: USAGE.md [hunk 2]
```diff
@@ -491,6 +491,33 @@ Make sure that your CNFs containers are not sharing the same [database](https://
...
+
+##### To run the Sigterm Handled test, you can use the following command:
+
+```
+./cnf-testsuite sig_term_handled
+```
+
+<b>Remediation for failing this test:</b>
+
+Make the the PID 1 container process to handle SIGTERM, enable process namespace sharing in Kubernetes or use specialized Init system.
```
Comment (agentpoyo) [inline USAGE.md:504]:
```suggestion
Make the PID 1 container process to handle SIGTERM; enable process namespace sharing in Kubernetes or use specialized Init system.
```

File: USAGE.md [hunk 3]
```diff
@@ -491,6 +491,33 @@ Make sure that your CNFs containers are not sharing the same [database](https://
...
+
+##### To run the Zombie Handled test, you can use the following command:
+
+```
+./cnf-testsuite zombie_handled
+```
+
+<b>Remediation for failing this test:</b>
+
+Make the the PID 1 container process to handle/reap zombie processes, enable process namespace sharing in Kubernetes or use specialized Init system.
```
Comment (agentpoyo) [inline USAGE.md:517]:
```suggestion
Make the PID 1 container process to handle/reap zombie processes; enable process namespace sharing in Kubernetes or use specialized Init system.
```

File: docs/LIST_OF_TESTS.md [hunk 4]
```diff
@@ -209,6 +209,20 @@ You can read more about horizonal pod autoscaling to create replicas [here](http
 
 [**Rationale & Reasoning**](../RATIONALE.md#to-check-if-the-cnf-uses-a-shared-database-shared_database)
 
+## [Sigterm Handled](https://github.com/cnti-testcatalog/testsuite/blob/v0.46.0/src/tasks/workload/microservice.cr#L500)
+- Expectation: Sigterm is handled by PID 1 process of containers.
+
+**What's tested:** This tests if PID 1 process of containers handles SIGTERM.
```
Comment (agentpoyo) [inline docs/LIST_OF_TESTS.md:215]:
```suggestion
**What's tested:** This tests if the PID 1 process of containers handles SIGTERM.
```

File: docs/LIST_OF_TESTS.md [hunk 5]
```diff
@@ -209,6 +209,20 @@ You can read more about horizonal pod autoscaling to create replicas [here](http
...
+
+[**Rationale & Reasoning**](../RATIONALE.md#to_check_if_the_cnf_pid_1_processes_handle_sigterm)
+
+## [Zombie Handled](https://github.com/cnti-testcatalog/testsuite/blob/v0.46.0/src/tasks/workload/microservice.cr#L436)
+- Expectation: Zombie processes are handled/reaped by PID 1 process of containers.
+
+**What's tested:** This tests if PID 1 process of containers handles/reaps zombie processes.
```
Comment (agentpoyo) [inline docs/LIST_OF_TESTS.md:222]:
```suggestion
**What's tested:** This tests if the PID 1 process of containers handles/reaps zombie processes.
```

**Review by agentpoyo — CHANGES_REQUESTED** [review]
Please review suggestions, these now flow better when reading.

**Review by lixuna — APPROVED** [review]
LGTM

**Review by agentpoyo — APPROVED** [review]
LGTM

**Pull request merged**